pkgs/top-level/all-packages.nix

```diff
@@ -1117,6 +1117,10 @@ with pkgs;
     pythonPackages = python2Packages;
   };
 
+  bfg-repo-cleaner = callPackage ../applications/version-management/git-and-tools/bfg-repo-cleaner {};
+
+  bfg = bfg-repo-cleaner;
```

Comment:
I don't agree with taking this attr in addition to the one above. Most packages get by with a single attr and I think that's sufficient here as well.